integration_testing/features/android/device-settings.feature

```diff
@@ -0,0 +1,103 @@
+		Given I have at least one unlisted channel
+		When I click the checkbox *Allow other computers on this network to import my unlisted channels*
+		Then I see that the checkbox is checked
+		When I click *Save changes*
+			And I try to import an unlisted channel using another computer in the same network
+		Then I can see the unlisted channel
+			And I can import the unlisted channel
+
+	Scenario: Set the default landing page to *Learn page*
+    When I select *Learn page* as the *Default landing page*
+    Then I see that the *Sign-in page* option is no longer selected
+    	And I see that the options *Allow users to access resources without signing in*, *Learners must sign-in to explore resources* and *Signed in learners should only see resources assigned to them in classes* are disabled (grayed out)
+    When I click the *Save changes* button
+      And I sign out
+    Then I see the *Learn > Library* page
+    When I sign-in as learner <username>
+    Then I also see the *Learn > Library* page
+
+  Scenario: Set the default landing page to *Sign-in page - Allow users to access resources without signing in*
+  	When I select *Sign-in page* as the *Default landing page*
+  	Then I see that the *Sign-in page* option is selected
+  		And I see that the *Allow users to access resources without signing in* option is also selected
+  	When I click the *Save changes* button
+      And I sign out
+    Then I see the *Sign-in* page
+    	And I see the *Explore without account* link
+    When I click the *Explore without account* link
+    Then I see the *Learn > Library* page
+
+  Scenario: Set the default landing page to *Sign-in page - Learners must sign in to explore resources*
+		Given I've selected the *Sign-in page* as the *Default landing page*
+		When I select the *Learners must sign-in to explore resources* option
+			And I click the *Save changes* button
+      And I sign out
+    Then I see the *Sign-in* page
+    	And I don't see the *Explore without account* link
+
+  Scenario: Set the default landing page to *Sign-in page - Signed in learners should only see resources assigned to them in classes*
+  	Given I've selected the *Sign-in page* as the *Default landing page*
+  	When I select the *Signed in learners should only see resources assigned to them in classes* option
+			And I click the *Save changes* button
+      And I sign out
+    Then I see the *Sign-in* page
+    When I sign-in as learner <username>
+    Then I see the *Learn > Home* page
+      And I don't see the *Library* or *Bookmarks* tabs
+
+	Scenario: Allow download on metered connection
+		When I select the *Allow download on a metered connection* option
+			And I click the *Save changes* button
+		Then I see a *Settings have been updated* message #next steps to be further discussed
+
+	Scenario: Change the storage location
+		When I click the *Manage* link
+		Then I see the *Storage location* modal
+			And I see that the current path is already selected
```

Comment:
This has been changed so that the current path and non-selectable paths are displayed outside the modal. Only the paths that can be set as the primary storage location should display in the modal